### ISSUE TITLE
Icon errors

### ISSUE THREAD

**Issue by Damillora** [themachinist/DiscourseSkin#2]
This is the error I have said in the pull request
I have discovered errors, related to icon_class.

Screenshot:
![asafsfasd](https://cloud.githubusercontent.com/assets/5674601/3948942/926312ce-26ae-11e4-9ca7-1980845957bd.PNG)

This one is related to the Echo extension:
![sgdssdg](https://cloud.githubusercontent.com/assets/5674601/3948943/9293e098-26ae-11e4-9bd9-7f2e8ffe6c57.PNG)
Can you figure out the solution?

**Comment by themachinist** [comment]
yes, i will make a revision. in this instance it isn't checking to see if the array key exists prior to trying to use it. that was a poor decision on my part

``` php
    echo '<ul class="nav nav-pills" id="navigation-bar">';
        foreach ( $value as $views ){
            if ( !is_null($views) ){
            // whoops:
                echo <<<HTML
                <li class="{$views['class']}" title="{$views['text']}">
                    <a href="{$views['href']}"><i class="fa {$views['icon_class']}"></i>{$views['text']}</a>
                </li>
HTML;
            }
        }
    echo '</ul>';
```

in the mean time, you could disable reporting of E_NOTICE level errors: 

``` php
error_reporting(E_ALL & ~E_NOTICE);
```
